js/main.js

```diff
@@ -183,6 +184,12 @@ requirejs(['bootstrap'], function () { // bootstrap must come first
 		var pageModel = new app();
 		window.pageModel = pageModel;
 
+		const originalTextBinding = ko.bindingHandlers.text.update;
```

Comment:
Not sure if this is needed for this set of changes? Or could this be related to the XSS changes?